components/uid.rst

```diff
@@ -27,48 +27,106 @@ Generating UUIDs
 ~~~~~~~~~~~~~~~~
 
 Use the named constructors of the ``Uuid`` class or any of the specific classes
-to create each type of UUID::
+to create each type of UUID:
 
-    use Symfony\Component\Uid\Uuid;
+**UUID v1** (time-based)
+    Generates the UUID using a timestamp and the MAC address of your device
+    (`read UUIDv1 spec <https://datatracker.ietf.org/doc/html/draft-ietf-uuidrev-rfc4122bis#name-uuid-version-1>`__).
+    Both are obtained automatically, so you don't have to pass any constructor argument::
+
+        use Symfony\Component\Uid\Uuid;
+
+        // $uuid is an instance of Symfony\Component\Uid\UuidV1
+        $uuid = Uuid::v1();
+
+    .. tip::
+
+        It's recommended to use UUIDv7 instead of UUIDv1 because it provides
+        better entropy.
+
+**UUID v2** (DCE Security)
```

Comment:
Is it for `Distributed Computing Environment` ?
We can maybe add it to description ?

Reply:
I added a bit more of information ... but I don't want to include many details because UUIDv2 is not implemented and it's not recommended either. Thanks.